Makefile

```diff
@@ -40,3 +40,13 @@ oapigen: install-oapi-codegen
 	cd v2/oapi/; go generate
 	@rm v2/oapi/source.json
 	ls -l v2/oapi/oapi.gen.go
+
+.PHONY: generate
+generate:
+	@go install go.uber.org/mock/mockgen@latest
```

Comment:
Shouln't be needed here, as we decided that generating mocks is up to the consumer.